app/Models/Ustensil.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kami\Cocktail\Models;
+
+use Spatie\Sluggable\HasSlug;
+use Spatie\Sluggable\SlugOptions;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+
+class Ustensil extends Model
+{
+    use HasFactory, HasImages, HasSlug;
+
+    private string $appImagesDir = 'ustensils/';
+
+    public function getSlugOptions(): SlugOptions
```

Comment:
I don't think there is a need for slug options in this feature, you can remove this method